app/components/filters/date-range-filter.hbs

```diff
@@ -1,16 +1,20 @@
-<Filters::Filter @id={{@id}} @label={{@label}}>
-  <AuDateRangePicker
-    @width="block"
-    {{!template-lint-disable no-duplicate-id}}
-    id={{@id}}
-    @label={{@label}}
-    @start={{@valueA}}
-    @end={{@valueB}}
+<Filters::Filter @label={{@label}}>
+    <AuDateRangePicker
+      @width="block"
 
-    @cancelAction={{this.resetDate}}
-    @applyAction={{this.dateChange}}
-    @hideAction={{this.dateChange}}
+      id={{@id}}
+      @label={{@label}}
+      
+      @start={{this.start}}
+      @end={{this.end}}
+      
+      @cancelAction={{this.resetDate}}
+      @applyAction={{this.dateChange}}
+      @hideAction={{this.hideDate}}
+
+      @cancelLabel="Reset"
+    />
+    {{!-- did-insert didn't work on the custom components --}}
+    <span class="au-u-hidden" {{did-insert this.hideDate}}></span>
```

Comment:
It's a _very_ funky fix, and I'm pretty sure there must be better solutions, but I don't have the time to look into it and I think it's better spent elsewhere anyways. We just merge it as-is since it does fix some of the issues already (without introducing other ones on first sight 😄)?